ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -235,3 +235,15 @@ kubectl create ns default || true
 kubectl config set-context --current --namespace=default
 KATA_RUNTIME=kata-remote ./deploy_webhook.sh
 popd
+
+
+##################################
+# Log warning when peering created
+##################################
+if [[ "${AZURE_REGION}" != "${PP_REGION}" ]]; then
+	echo "This script created additional resources to create peering between ${AZURE_REGION} and ${PP_REGION}. Ensure you release those resources after the testing (or use temporary subscription)"
```

Comment:
```suggestion
	echo "This script created additional resources for peering between ${AZURE_REGION} and ${PP_REGION}. Ensure you release those resources after the testing (or use temporary subscription)"
```

Reply:
Is there a simple command we could list here to delete the resource?
Does the script create a resource group? If so maybe some `az group delete`?
Any other idea on what to suggest here?

Also, since this is a CI script, where does that message go? If nobody watches it because the CI was successful, the message may not end up being so successful at achieving its objective 😄

Reply:
Good idea, yes it's a separate group. In CI we don't care because temporary accounts are being used and they trash all resources on deletion. I'll add the az delete commands.